frontend/app/onboarding/onboarding-upgrade.tsx

```
@@ -0,0 +1,171 @@
+                                <i className="fa fa-sparkles" />
+                                <span className="font-bold ml-2 font-mono">Wave AI</span>
+                            </div>
+                            <div className="text-secondary leading-relaxed max-w-[420px]">
+                                <p className="mb-4">
+                                    Wave AI is your new terminal assistant with full context. It can read your terminal
+                                    output, analyze widgets, access files, and help you solve problems&nbsp;faster.
+                                </p>
+                                <p className="p-3 border border-border rounded-md bg-hover/30">
+                                    Wave AI is in active beta with included AI credits while we refine the experience.
+                                    We're actively improving it and would love your feedback in{" "}
+                                    <a target="_blank" href="https://discord.gg/XfvZ334gwU" className="hover:underline">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)